docs/book/orders.rst

```diff
@@ -35,6 +35,24 @@ Then get a channel to which you would like to add your Order. You can get it fro
 
     $order->setChannel($channel);
 
+Next give your order a locale.
+
+.. code-block:: php
+
+    /** @var LocaleInterface $locale */
+    $locale = $this->container->get('sylius.context.locale')->getLocale();
+
+    $order->setLocale($locale);
+
+And a currency:
+
+.. code-block:: php
+
+    /** @var CurrencyInterface $currency */
+    $Currency = $this->container->get('sylius.context.currency')->getCurrency();
+
+    $order->setCurrency($currency);
```

Comment:
Moreover this is wrong. Order doesn't have a `currency` object anymore, it's split between `currencyCode` and `exchangeRate`. Same goes with `locale` => `localeCode`.
Also, both currency and locale contexts return code atm, so `->get*Code()`.